docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  clickhouse:
+    image: 'clickhouse/clickhouse-server:${CLICKHOUSE_VERSION-24.10-alpine}'
```

Comment:
What about `head-alpine`?

Reply:
I'd say let's keep the static version for development purposes.
On the CI, it is preferable to have a matrix of at least `head` + `latest` plus maybe an LTS version.